lib/wrappers/angular.js

```diff
@@ -27,8 +27,14 @@
             }, angularAdapter);
 
             // Wrap `execute()` in an Angular promise
-            api.execute = function(method, params) {
-                return $q.when(TaggedApi.prototype.execute.call(this, method, params));
+            api.execute = function(method, options) {
+                if (!options.clientId || typeof options.clientId === 'undefined') {
```

Comment:
I think it would be better if this check was handled in the constructor. Once set, it doesn't need to change.